### PR TITLE
Use Alpine instead of Ubuntu as base images

### PR DESCRIPTION
**Pull request by emileswarts**
This also bumps Ruby from 2.5 to 2.6
It looks like kubectl is unused in this container by the application code, so
leaving this out.  The default port for Rails to boot up on is 3000, so no need to be explicit about this.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,17 @@
...
-RUN apt-get install -y kubectl
+RUN bundle install --jobs 4 --retry 5 --without test development
 
-# allow access to port 3000
-ENV APP_PORT 3000
-EXPOSE $APP_PORT
+RUN apk del .build-deps
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup
+RUN chown -R appuser:appgroup .
+USER appuser
```
Comment (asmega) [inline Dockerfile]: great if this works! I remember having issues with image not being able to be deployed if the user id was not over 1000 hopefully you won't run into the same issue

File: Gemfile [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-ruby '2.5.1'
+ruby File.read('.ruby-version').chomp
 
 gem 'rails', '~> 5.2.1', '>= 5.2.1.1'
-gem 'puma', '~> 3.11'
+gem 'puma', '~> 3.12'
 gem 'aws-sdk-s3', '~> 1'
 gem 'jwt'
 gem 'sentry-raven'
+gem 'tzinfo-data'
```
Comment (asmega) [inline Gemfile]: this gem is specified twice in the Gemfile

File: Dockerfile (hunk 1 above)
Comment (emileswarts) [reply Dockerfile]: Oh I see.  Yeah not had any issues deploying this out.

File: Gemfile (hunk 2 above)
Comment (emileswarts) [reply Gemfile]: Cool have removed it

File: Dockerfile (hunk 1 above)
Comment (tommotaylor) [reply Dockerfile]: You were right @asmega

**Comment by tommotaylor** [comment]
UPDATE: This is passing all specs locally but when the test suite is run inside docker there are numerous failures.

**Pull request merged**